chia/cmds/wallet_funcs.py

```diff
@@ -828,6 +829,79 @@ async def get_did(args: Dict, wallet_client: WalletRpcClient, fingerprint: int)
         print(f"Failed to get DID: {e}")
 
 
+async def get_did_info(args: Dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    coin_id: str = args["coin_id"]
+    latest: bool = args["latest"]
+    try:
+        response = await wallet_client.get_did_info(coin_id, latest)
+        print(f"{'DID:'.ljust(23)} {response['did_id']}")
+        print(f"{'Coin ID:'.ljust(23)} {response['latest_coin']}")
+        print(f"{'Inner P2 Address:'.ljust(23)} {response['p2_address']}")
+        print(f"{'Public Key:'.ljust(23)} {response['public_key']}")
+        print(f"{'Launcher ID:'.ljust(23)} {response['launcher_id']}")
+        print(f"{'DID Metadata:'.ljust(23)} {response['metadata']}")
+        print(f"{'Recovery List Hash:'.ljust(23)} {response['recovery_list_hash']}")
+        print(f"{'Recovery Required Verifications:'.ljust(23)} {response['num_verification']}")
+        print(f"{'Last Spend Puzzle:'.ljust(23)} {response['full_puzzle']}")
+        print(f"{'Last Spend Solution:'.ljust(23)} {response['solution']}")
+        print(f"{'Last Spend Hints:'.ljust(23)} {response['hints']}")
```

Comment:
A tiny nit: all these `23` should probably be a named constant, in case we need to change it. But also, just adding the spaces to the format string might make it easier to read